.github/workflows/manifold.yml

```diff
@@ -100,6 +100,15 @@ jobs:
       run: |
         cd build/test
         node ./manifold_test.js
+    - name: 'Upload WASM files'
+      uses: actions/upload-artifact@v3
+      with:
+        name: demo
+        path: |
+          build/bindings/wasm/index.html
```

Comment:
I don't think we need to upload this, since it's already served by `deploy.sh`: see here (the page is blank, since it can't find `manifold.js`): https://elalish.github.io/manifold/bindings/wasm/examples/index.html